### DISCUSSION TITLE
prevent user from moving object on y axis

### DISCUSSION THREAD

**Discussion by veronicamontezuma** [google/model-viewer#2120] [Q&A] [answered]
Hi there!
I was wondering if someone could help me – I have a model placed on the wall (webXR/quicklook), but I don't want the user to be able to change it's "height" from the floor. Is there a way I can prevent this?
I searched the docs but nothing seemed do to that, looks like only the scaling is something I can stop. Is that so?

**Comment by elalish** [comment, marked as answer]
No, we don't have a way to control this. What is your use case?

**Reply by veronicamontezuma** [reply]
it's an artwork model that should be placed at a specific height, and not manipulated beyond initial positioning (ideally). Is it possible to prevent touch gestures entirely? that could sort of solve it.

**Reply by elalish** [reply]
I think you might be better off with floor placement instead of wall placement. I believe you can use an animation translation to achieve a vertical offset from the floor. If you register touch handlers and call `preventDefault()` I believe you'll cancel out our UX.

**Reply by barzorke** [reply]
preventDefault() doesn't seams to block camera controle :(

**Reply by elalish** [reply]
@barzorke If you provide a repro URL I'll take a look. Maybe you need  to prevent  propagation?
